src/client/ui/selector-inspector-panel/pick-button.js

```diff
@@ -0,0 +1,18 @@
+import testCafeCore from './../deps/testcafe-core';
+
+import { createElementFromDescriptor } from './utils/create-element-from-descriptor';
+
+import { pickButton } from './descriptors';
+import { elementPicker } from './element-picker';
+
+const eventUtils = testCafeCore.eventUtils;
+
+export class PickButton {
+    element;
```

Comment:
same thing. We we need this. It's not typescript. The constructor is enough, isn't it?